### ISSUE TITLE
Would a SnapCraft package for Helm be possible?

### ISSUE THREAD

**Issue by css459** [mtytel/helm#219]
I just want to start off saying I've really enjoyed this synth and I think it's really well done! I currently use a Linux OS called Solus, and was wondering if I can use an installation option outside of `make install`. 

I was working on a package specification for Solus, however, I later learned that this was requested earlier but because of certain licensing issues with JUCE, it will not be included ([source](https://dev.getsol.us/T6358)). 

Would packaging Helm into a Snap package on [snapcraft.io](https://snapcraft.io/ ) be possible? I am willing to help with this, if you would like. Also, I can send
[package.zip](https://github.com/mtytel/helm/files/2753568/package.zip)
 you the `package.yml` specification for use with Solus's package manager if you are interested, or would like to build a `.eopkg` bundle to host on your site. 

Thanks again for the great piece of software!

**Comment by mtytel** [comment]
Seems to be a misunderstanding in the JUCE license.
JUCE has a dual license where you pick GPLv3 or their proprietary license and I think they were reading through the proprietary EULA. I commented on the solus page so hopefully that clears it up and Helm can go into the official repositories.

**Comment by css459** [comment]
Sounds good! I'll willing to test build packaging on my system if that will help. Here is a `package.yml` I wrote that compiles on my system: 

```
name       : helm
version    : 0.9.0
release    : 1
source     :
    # SHA256SUM is latest commit hash when this file was created.
    # A hosted tar ball is ideal
    - https://github.com/mtytel/helm.git : abdedd527e6e1cf86636f0f1e8a3e75b06ed166a
license    : GPL-3.0-or-later
summary    : Polyphonic synthesizer with realtime modulation feedback
component  : multimedia.audio
description: |
    Helm is a MIDI enabled polyphonic music synthesizer with an easy to use
    parameter modulation system with real-time graphical feedback.
    Helm has a subtractive synthesizer engine with three oscillators with 15
    anti-aliased waveforms, cross modulation, 32 voice polyphony, a low-pass,
    high-pass, band-pass, notch, low-shelf, high-shelf and all-pass filter.
    There is also a formant filter, stutter, delay and reverb effects,
    patch browsing, MIDI learn and more.
rundeps    :
    - alsa-lib
    - freetype2
    - libgcc
    - libglu
    - libstdc++
    - libx11
    - libxext
builddeps  :
    - pkgconfig(alsa)
    - pkgconfig(libcurl)
    - pkgconfig(xinerama)
    - pkgconfig(jack)
    - pkgconfig(xcursor)
setup      : |
    %configure
build      : |
    %make
install    : |
    %make_install
```

**Comment by css459** [comment]
Additionally, the Solus developers denied inclusion of a pre-release package (as seen in https://github.com/mtytel/helm/releases). Is a full release coming soon?

**Comment by mtytel** [comment]
Yeah probably this summer. I'm working on a new synth that will pretty much replace Helm but I'll have some time this summer to bump the version number and fix a few last bugs.
